main/src/ca/uwaterloo/flix/language/phase/unification/BddFormula.scala

```diff
@@ -15,10 +15,166 @@
  */
 package ca.uwaterloo.flix.language.phase.unification
```

Comment:
I would like to move towards merging this :)

This means we (OK you :)) should spend some time polishing. Ill try to give some directions!